qiskit/transpiler/passes/synthesis/unitary_synthesis.py

```diff
@@ -725,7 +725,9 @@ def run(self, unitary, **options):
 
         if unitary.shape == (2, 2):
             _decomposer1q = Optimize1qGatesDecomposition(basis_gates, target)
-            return _decomposer1q._resynthesize_run(unitary, qubits[0])  # already in dag format
+            return _decomposer1q._gate_sequence_to_dag(
+                _decomposer1q._resynthesize_run(unitary, qubits[0])
+            )  # already in dag format
```

Comment:
This comment looks out-of-date now?

Reply:
Dropped in: https://github.com/Qiskit/qiskit-terra/pull/9578/commits/b8992a7dc3f22c1cb27e475c82165d41ddee511a